src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs

```diff
@@ -48,17 +48,51 @@ public TransferConnectionInvitationRepository(EmployerApprenticeshipsServiceConf
                 var parameters = new DynamicParameters();
 
                 parameters.Add("@id", id, DbType.Int64);
-                parameters.Add("@status", TransferConnectionInvitationStatus.Sent, DbType.Int16);
+                parameters.Add("@status", TransferConnectionInvitationStatus.Pending, DbType.Int16);
+
+                const string sql = @"
+                    SELECT i.*, r.name AS ReceiverAccountName
```

Comment:
This code cannot guarantee the order or layout of the TransferConnectionInvitation table. The i.* makes the code sensitive to schema changes.